Terminal.Gui/Views/ColorPicker.cs

```diff
@@ -0,0 +1,194 @@
+﻿//
+// ColorPicker.cs: ColorPicker control
+//
+// Authors:
```

Comment:
Please remove the Authors section. We use github to track who contributed what and prefer to not have the information duplicated within the source.

Reply:
Done